mozilla_django_oidc/utils.py

```diff
@@ -51,3 +57,46 @@ def is_authenticated(user):
     msg = '`is_authenticated()` is going to be removed in mozilla-django-oidc v 2.x'
     warnings.warn(msg, DeprecationWarning)
     return user.is_authenticated
+
+
+def add_state_and_nonce_to_session(request, state, params):
```

Comment:
Could you add a docstring in this method or move the comments below here as a docstring?

Reply:
Done.